### ISSUE TITLE
Convert from XCF to TIF fails for latest release

### ISSUE THREAD

**Issue by jchnkl** [ImageMagick/ImageMagick#1635]
### Prerequisites

- [x] I have written a descriptive issue title
- [x] I have verified that I am using the latest version of ImageMagick
- [x] I have searched [open](https://github.com/ImageMagick/ImageMagick/issues) and [closed](https://github.com/ImageMagick/ImageMagick/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

There are issues concerning this topic, but only for older versions.

### Description
`$ ./ImageMagick-4907e6f-gcc-x86_64.AppImage convert 0001.{xcf,tif}`
convert: data storage type is not supported \`0001.xcf' @ error/xcf.c/ReadXCFImage/1194.
convert: no images defined \`0001.tif' @ error/convert.c/ConvertImageCommand/3273.

### Steps to Reproduce
see above

### System Configuration
Ubuntu 18.04
ImageMagick Release build (7.0.8-53)

**Comment by fmw42** [comment]
I do not believe that Imagemagick currently supports the latest versions of GIMP xcf format. Not sure why tif fails for you. Do you have libtif installed as a delegate?  What do you get from `magick -version`. Note for IM 7, use magick not convert.  What was your exact full command line? Perhaps the tif error is ancillary due to unsupported xcf format.

**Comment by jchnkl** [comment]
```
$ ./ImageMagick-4907e6f-gcc-x86_64.AppImage -version
Version: ImageMagick 7.0.8-53 Q16 x86_64 2019-07-06 https://imagemagick.org
Copyright: © 1999-2019 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Features: Cipher DPC 
Delegates (built-in): bzlib djvu fontconfig freetype gvc jbig jng jpeg lcms lqr lzma openexr png raqm tiff wmf x xml zlib
```

> I do not believe that Imagemagick currently supports the latest versions of GIMP xcf format.

I think that might be the answer here.

> Do you have libtif installed as a delegate?

From what I can see in the above output it's built into the AppImage.

> What was your exact full command line?

As I wrote under `Description` in my initial bug report:

`$ ./ImageMagick-4907e6f-gcc-x86_64.AppImage convert 0001.{xcf,tif}`

I tried two more times:

```
$ /tmp/ImageMagick-4907e6f-gcc-x86_64.AppImage 0001.xcf /tmp/0001.tif
magick: data storage type is not supported `0001.xcf' @ error/xcf.c/ReadXCFImage/1194.

$ /tmp/ImageMagick-4907e6f-gcc-x86_64.AppImage convert 0001.xcf /tmp/0001.tif
convert: data storage type is not supported `0001.xcf' @ error/xcf.c/ReadXCFImage/1194.
convert: no images defined `/tmp/0001.tif' @ error/convert.c/ConvertImageCommand/3273.
```

**Comment by dlemstra** [comment]
Can you still reproduce this issue? And if you can could you please share your input image?
